frontend/src/components/SchemaForm/SchemaFormInput.tsx

```diff
@@ -55,11 +55,11 @@ const SchemaFormInput = ({ field, value, readOnly }: SchemaFormInputProps) => {
         />
       );
     case "SingleSelect":
-      const choices = field.choices ?? [];
+      let choices = field.choices ?? [];
 
       switch (field.presentation) {
         case "dropdown":
-          choices.unshift({ slug: "", title: "" });
+          choices = [{ slug: "", title: "" }].concat(field.choices);
```

Comment:
there's a splat for a more civilized age
```suggestion
          choices = [{ slug: "", title: "" }, ...field.choices];
```